src/unstract/adapters/embedding/ollama/src/static/json_schema.json

```diff
@@ -3,6 +3,7 @@
   "type": "object",
   "required": [
     "adapter_name",
+    "base_url",
```

Comment:
@gaya3-zipstack if this is made required, we might need a migration for existing adapters

Reply:
@chandrasekharan-zipstack  Ollama embedding has only reached staging and so far @VikashPratheepan has not been able to create any adapter successfully with it as this param was not configurable.

Reply:
Also I believe without the base_url users anyway wouldn't have been able to add this adapter.